src/header.wppl

```diff
@@ -531,7 +531,7 @@ var Vector = function(arr) {
 var Matrix = function(arr) {
   var n = arr.length;
   var m = arr[0].length;
-  var t = ad.tensor.fromScalars(_.flatten(arr));
+  var t = ad.tensor.fromScalars(_.flattenDeep(arr));
```

Comment:
This was to be paranoid/safe when migrating. `flatten` in underscore is deep unless a shallow argument passed, while lodash breaks them up into two functions, `flatten` which is shallow and `flattenDeep` which is deep.  I guess if the matrix is always 2D, I can change it.